test/functional/Java8andUp/src/org/openj9/test/java/lang/reflect/Test_Executable.java

```diff
@@ -351,10 +359,21 @@ public void test_getParameters() {
 			}
 
 			checkDeclaringExecutable(parameters[0], methods[1]);
-			checkParameterName(methods[1], parameters[0], "name");
-			checkParameterModifiers(methods[1], parameters[0], ACC_MANDATED);
+			checkParameterName(methods[1], parameters[0], "arg0");
+			checkParameterModifiers(methods[1], parameters[0], ACC_DEFAULT);
```

Comment:
why ACC_DEFAULT instead of ACC_MANDATED?

Reply:
@DanHeidinga When the code is compiled without `-parameters`, getModifiers() on the parameters returns 0. See the other tests without `-parameters`. The only reason it worked with ACC_MANDATED before is because the "without parameters" test was using the wrong "with parameters" class.